### DISCUSSION TITLE
Ground Items Plugin Exception

### DISCUSSION THREAD

**Discussion by Looodon** [runelite/runelite#17946] [Ideas]
Hey! It would be nice if I can add exceptions to the "Hide Under Value" filter in the Ground Items plugin. For example, when i want an item to show that's under value, it doesn't show up because I have "hide under value" at 1500 so I have to resort to adding said item into my highlighted items section, but the problem with that is there's would an alarm notification every kill. It would be nice for the item to show up but not as a highlighted item. Thanks!

**Comment by Trencha** [comment]
Just want to throw another voice behind this.

There are a couple of other discussion threads where this (or something similar) has been suggested:

#17159 
#17370 
#15527 

I also recently came across my own use case for this feature.

I have the following settings for the Ground Items plugin:

- Hide under value 400
- Deprioritize hidden items
- Notify for highlighted drops

Due to my settings, I am unable to pick up the items that spawn when a Hunter trap fails (e.g. rope and small fishing net for a net trap) without right-clicking, which is annoying.

There are two possible workarounds that I have found for this.

The first is to change my settings for the plugin on the fly when I am training Hunter. This is clunky and requires me to remember what my settings were before I changed them and remember to change them back afterwards.

The second (and the one that I am using right now) is to add the items used in the trap (in my case, rope and small fishing net) to the highlighted items so that they don't get hidden. This results in me getting two notifications every time a trap fails (one for the rope and one for the net), and also getting a notification whenever I happen to enter an area that has a rope or small fishing net spawn on the ground. Also, I don't think it's really what highlighting items is intended for.

It would be better if I could just add rope and small fishing net to some kind of whitelist that ignores the setting to hide items below a specified value.

**Reply by cdfisher** [reply]
Loot Filters on the Plugin Hub might be able to do what you're after.

**Reply by riktenx** [reply]
(it can)
